demisto_sdk/__main__.py

```diff
@@ -2894,8 +2894,8 @@ def test_content(ctx, **kwargs):
     default=False,
 )
 @click.option(
-    "-pkw",
-    "--use-packs-known-words",
+    "-pkw/ -npkw",
+    "--use-packs-known-words/ --no-use-packs-known-words",
```

Comment:
```suggestion
    "--use-packs-known-words/--skip-packs-known-words",
```
please change elsewhere as well

Reply:
Done